example_test.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials_test
+
+import (
+	"context"
+	"fmt"
+
+	credentials "github.com/oras-project/oras-credentials-go"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func ExampleNewStoreFromDocker() {
+	ds, err := credentials.NewStoreFromDocker(credentials.StoreOptions{AllowPlaintextPut: true})
```

Comment:
Let's format the code in standard go style.
```suggestion
	ds, err := credentials.NewStoreFromDocker(credentials.StoreOptions{
		AllowPlaintextPut: true,
	})
```

Reply:
same for the other example